packages/app/src/pages/connect/index.ts

```diff
@@ -0,0 +1 @@
+export * from './choose-account';
```

Comment:
Why this? It looks like we have one import for `pages/connect` - lets just move this to import `pages/choose-account`

Reply:
That's fine, I was thinking it would make sense for our folder structure to match the paths, but happy for either to work.